prow/apis/prowjobs/v1/types.go

```diff
@@ -532,7 +532,10 @@ func (g *GCSConfiguration) Validate() error {
 
 // ProwJobStatus provides runtime metadata, such as when it finished, whether it is running, etc.
 type ProwJobStatus struct {
-	StartTime      metav1.Time  `json:"startTime,omitempty"`
+	StartTime metav1.Time `json:"startTime,omitempty"`
```

Comment:
nit: could we add godoc here, is `StartTime` ever different from `ProwJob.Metadata.CreationTimestamp`?

Reply:
Done.